sunpy/visualization/colormaps/color_tables.py

```diff
@@ -68,20 +69,22 @@ def paden(lst, len_, pad=0):
 c3 = ((c1 + c2 / 2.0) * 255.0 / (c1.max() + c2.max() / 2.0)).astype('f')
 
 aia_wave_dict = {
-    1600: (c3, c3, c2),
-    1700: (c1, c0, c0),
-    4500: (c0, c0, b0 / 2.0),
-    94: (c2, c3, c0),
-    131: (g0, r0, r0),
-    171: (r0, c0, b0),
-    193: (c1, c0, c2),
-    211: (c1, c0, c3),
-    304: (r0, g0, b0),
-    335: (c2, c0, c1)
+    1600: (c3, c3, c2)*u.angstrom,
```

Comment:
```suggestion
    1600*u.angstrom: (c3, c3, c2),
```
Like that and then in theory we can index the dict with astropy units. Will need to test this to make sure.

Reply:
You can in fact use quantities as dictionary keys 🚀